option optimize_for = LITE_RUNTIME;

message GallantEntryActivateRequest
{
 required uint32 id = 1;
}
